Para refrescar la memoria, recordemos que mensajes le habíamos enviado:

```python
ムcelular_de_eli.necesita_saldo()
False
ムcelular_de_eli.cargar_bateria()
File "<input>", line 1, in <module>
AttributeError: 'Celular' object has no attribute 'cargar_bateria'
ムcelular_de_eli.tiene_bateria_maxima()
False
ムcelular_de_eli.cargar_a_tope()
```
Tené en cuenta, que si un objeto no entiende un mensaje la consola nos mostrará un error. :confounded: